### ISSUE TITLE
upload a list of locations

### ISSUE THREAD

**Issue by markherringer** [healthsites/healthsites#521]
as an administrator I want to upload a list of locations to the site.
the feature should be available from /admin

This feature has been developed by @ismailsunni and includes a report

Please see other tickets:
#309
#210
#200
#149
#432
#204

**Comment by meomancer** [comment]
@markherringer 
is it related with : http://staging.healthsites.io/upload-form ?
because t is already done.
where should i put link in page?
(note, i will make restriction if user is admin)

**Comment by markherringer** [comment]
Hi @meomancer thank you for this. The idea for this feature is to allow trusted users like The ICRC to upload lists them selves. We need the ICRC user to do this so that the provenance is established.The process will work like this: 
1) The user will contact us 
2) we will help prepare the list for upload and terst on staging 
3) Once the list is ready we will assign the list upload permissions to the ICRC user so that they can upload it themselves
